pkgs/development/libraries/zeroc-ice/3.6.nix

```diff
@@ -27,13 +28,8 @@ stdenv.mkDerivation rec {
   '';
 
   patches = [
-    # Fixes compilation issues with GCC 8 using one of the patches
-    # provided in https://github.com/zeroc-ice/ice/issues/82
-    ( fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
good catch; thanks